fix: Vesting token display corrected

jay-dee7 opened the pull request:
Add: correct vesting information
Fix: cheq Amount in txn table

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

jay-dee7 committed:
Fix: vesting tokens calculation

Add: correct vesting information
Fix: cheq Amount in txn table

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

jay-dee7 committed:
Fix: vesting tokens calculation

Add: correct vesting information
Fix: cheq Amount in txn table

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

ankurdotb reviewed (changes requested):
Filed details on ClickUp on why part of the display is still incorrect

jay-dee7 committed:
Fix: vesting calculation integer overflow

We were using a lot of parseInt methods in estimatedVesting method and
it was leading to integer overflow for large numbers. We've switched
from parsing ints of `Coin.amount` to just type casting the amount
(which is a string) to Number in TS, which seems to fix the issue as of
now.

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

jay-dee7 committed:
Fix: vesting calculation integer overflow

We were using a lot of parseInt methods in estimatedVesting method and
it was leading to integer overflow for large numbers. We've switched
from parsing ints of `Coin.amount` to just type casting the amount
(which is a string) to Number in TS, which seems to fix the issue as of
now.

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

ankurdotb committed:
Updated Yarn

ankurdotb committed:
Update yarn lock

jay-dee7 committed:
fix: linting issues in account.ts

Signed-off-by: jay-dee7 <jasdeepsingh.uppal@gmail.com>

ankurdotb committed:
Update branches

ankurdotb committed:
Fixed vesting amount calculations

ankurdotb committed:
Update API endpoints

ankurdotb committed:
remove submodule

ankurdotb committed:
Added git submodule

ankurdotb committed:
Update .gitmodules

ankurdotb committed:
Update frontend-elements
